### PR TITLE
Alphabetize the plugins on the admin page.

### PR DESCRIPTION
**Pull request by manthey**
Plugins were in no consistent order, and they could get shuffled by the addition of an additional plugin.

### CODE REVIEW

File: clients/web/src/templates/body/plugins.jade [hunk 1]
```diff
@@ -8,17 +8,17 @@ div.g-plugin-restart
...
       .g-plugin-name
-        | #{plugin.name}
-        if (plugin.version)
+        | #{plugin.value.name}
+        if (plugin.value.version)
           span.g-plugin-version
-            | Version #{plugin.version)
-        if(plugin.configRoute)
-          a.g-plugin-config-link(g-route="#{plugin.configRoute}",
+            | Version #{plugin.value.version)
```
Comment (zachmullen) [inline clients/web/src/templates/body/plugins.jade]: This `#{...)` notation seems to work, but looks wonky to me, can we use a closing curly brace?

Comment (manthey) [inline clients/web/src/templates/body/plugins.jade]: That IS wonky.  A typo from when I first added it that peculiarly did no harm.  I'll fix it.

File: clients/web/src/views/body/PluginsView.js [hunk 2]
```diff
@@ -79,6 +79,23 @@ girder.views.PluginsView = girder.View.extend({
         return this;
     },
 
+    _sortPlugins: function (plugins) {
+        /* Sort a dictionary of plugins alphabetically so that the appear in a
+         * predictable order to the user.
+         *
+         * :param plugins: a dictionary to sort.  Each entry has a .name
+         *                 attribute used for sorting.
+         * :returns sortedPlugins: the sorted list. */
```
Comment (zachmullen) [inline clients/web/src/views/body/PluginsView.js]: Let's use jsdoc conventions in the javascript (@param, @returns)

**Comment by zachmullen** [comment]
LGTM

**Pull request merged**